documentation/docs/feedback/route-announcer.mdx

```diff
@@ -33,7 +33,7 @@ These are:
 - Start your screen-reader (like `VoiceOver` or `Narrator`)
 - Try navigating to any page visible on the demo.
 - The screen reader will announce any page changes, like "Navigated to The Best Excuses To Spend A Cozy Valentine’s Day In".
-:::
+  :::
```

Comment:
ditto, is this necessary? if not i think one less change could improve at least a bit of readability of the whole MR.